pkgs/top-level/all-packages.nix

```
@@ -9290,8 +9290,7 @@ in
 
   wsmancli = callPackage ../tools/system/wsmancli {};
 
-  wstunnel = haskell.lib.justStaticExecutables
-    (haskellPackages.callPackage ../tools/networking/wstunnel {});
+  wstunnel = throw "wstunnel has been removed in favor of haskellPackages.wstunnel";
```

Let's provide `wstunnel` in a backwards compatible way. Additionally this will require less downloading than the version from `haskellPackages` thanks to `justStaticExecutables` which removes the need to download unnecessary dependencies.

```suggestion
  wstunnel = haskell.lib.justStaticExecutables haskellPackages.wstunnel;
```